training-maquetado/login.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body>
+    <div class="login">
+        <img src="assets/logo-wolox.png" class="login-logo" alt="Wolox logo" />
+        <div class="input-box">
+            <label class="text-xsmall">Email</label>
+            <input class="login-input" type="email"></input>
+        </div>
+        <div class="input-box">
+            <label class="text-xsmall">Password</label>
+            <input class="login-input" type="password"></input>
+        </div>
+        <button class="login-button text-xsmall" onClick="window.location='book-list.html';">Login</button>
```

Comment:
mmm this onClick is weird.. why? i think this is a button type ='submit'

wrap the elements in a form and it will do the job